projects/RabbitMQ.Client/client/impl/Connection.cs

```diff
@@ -416,9 +421,66 @@ internal void Write(ReadOnlyMemory<byte> memory)
 
         internal ValueTask WriteAsync(ReadOnlyMemory<byte> memory)
         {
+            SetNetworkTags(Activity.Current);
             return _frameHandler.WriteAsync(memory);
         }
 
+        private void SetNetworkTags(Activity? activity)
+        {
+            if (activity != null && activity.IsAllDataRequested)
+            {
+                activity.SetTag("network.transport", "tcp");
```

Comment:
I think AMQP of any version always uses TCP as a transport. If so, we can probably remove the tag as a non-informative one. We can also document the default value in the rabbit semconv,

Reply:
This is correct, this client supports nothing but TCP for transport.

Reply:
AFAIK semcon changes schema supports rename and split only for breaking changes (types). If that is fixed as a constant in semcon and should actually change back to dynamic (in any language), it will be a hard breaking change for a back-end to handle.
Due limited supported functions in the changes schema, I guess it still happens from time to time. Just FYI.

Reply:
Removed for now